MSA-635: Sensibo integration

EricG66 opened the pull request:
Submitting 'Sensibo integration' for publication

EricG66 committed:
Initial version

EricG66 committed:
Initial version

EricG66 committed:
MSA-635: Control Sensibo Air - Conditioning with Smartthings.

See Sensibo at https://www.sensibo.com/

EricG66 committed:
MSA-635: Control Sensibo Air - Conditioning with Smartthings.

See Sensibo at https://www.sensibo.com/

Fixed Icons

Fixed some issues

Updated background color for the On/Off

Updated range value for setTemperaturePoint

Moved ServerUrl from AppSettings to in-code

Updated some files

Fixed event get called twice on installed and updated events

Fixed small event issue

Updated background color for on/off

Remove VALUE_CONTROL since it crash updated version in IOS

Added up/down control

Fixed bugs

Fixed raiseSetPoint issue

Incorporate temperature capability

Corrected some issues with scheduling

Fixed up/down button

Fixed another issue with up/down buttons

Passing the temperatureScale from the location to the device

fixed small bug

fixed small bug

Made changes in the events fired

Made clean up in the code + fixed events

Removed debugevents

Update some events

Made some cleanup

Fixed some colors

Add ability to receive push notification for the sensors

Simplified the SmartTiles

Added the abillity to limit on specific day and time to receive push notification

Fixed push notification and change some icons

Added limit Push notification

updated notification

Fixed notification

Fixed notification

Support for farenheit (reading from location Temperature Scale

Support for Farenheit (reading from Location)

Removed some commented code

EricG66 committed:
Fixed Icons

EricG66 committed:
Fixed some issues

EricG66 committed:
Updated background color for the On/Off

EricG66 committed:
Updated range value for setTemperaturePoint

EricG66 committed:
Moved ServerUrl from AppSettings to in-code

EricG66 committed:
Updated some files

EricG66 committed:
Fixed event get called twice on installed and updated events

EricG66 committed:
Fixed small event issue

EricG66 committed:
Updated background color for on/off

EricG66 committed:
Remove VALUE_CONTROL since it crash updated version in IOS

EricG66 committed:
Added up/down control

EricG66 committed:
Fixed bugs

EricG66 committed:
Fixed raiseSetPoint issue

EricG66 committed:
Incorporate temperature capability

EricG66 committed:
Corrected some issues with scheduling

EricG66 committed:
Fixed up/down button

EricG66 committed:
Fixed another issue with up/down buttons

EricG66 committed:
Passing the temperatureScale from the location to the device

EricG66 committed:
fixed small bug

EricG66 committed:
fixed small bug

EricG66 committed:
Made changes in the events fired

EricG66 committed:
Made clean up in the code + fixed events

EricG66 committed:
Removed debugevents

EricG66 committed:
Update some events

EricG66 committed:
Made some cleanup

EricG66 committed:
Fixed some colors

EricG66 committed:
Add ability to receive push notification for the sensors

EricG66 commented:
Better version with PushNotification

EricG66 committed:
Simplified the SmartTiles

EricG66 committed:
Added the abillity to limit on specific day and time to receive push notification

EricG66 committed:
Fixed push notification and change some icons

EricG66 committed:
Added limit Push notification

EricG66 committed:
updated notification

EricG66 committed:
Many updates

EricG66 committed:
Many updates

EricG66 committed:
Removed commented lines

EricG66 committed:
Removed commented lines

EricG66 committed:
Remove a buggy feature, will fix it before commit it again

EricG66 committed:
Fix some issues

EricG66 committed:
Fix some issues

EricG66 committed:
Fixed notification

EricG66 committed:
Fixed notification

EricG66 committed:
Support for farenheit (reading from location Temperature Scale

EricG66 committed:
Support for Farenheit (reading from Location)

EricG66 committed:
Removed some commented code

EricG66 committed:
Added support for Farenheit (reading from the Location Temperature Scale)

EricG66 committed:
Added support for Farenheit (reading from the Location Temperature Scale)

kris-schaller commented:
@EricG66 , can you please squash your commits into one to keep the history clean?

Thanks!

EricG66 commented:
Sure I will do. Just need to find out how ;)

Eric
Le 2015-11-10 17:10, "Kristofer Schaller" notifications@github.com a
écrit :

> @EricG66 https://github.com/EricG66 , can you please squash your
> commits into one to keep the history clean?
> 
> Thanks!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/213#issuecomment-155583975
> .

egosselin69 committed:
Merge remote-tracking branch 'refs/remotes/SmartThingsCommunity/master' into MSA-635-1

egosselin69 committed:
Merge branch 'MSA-635-1' of https://github.com/EricG66/SmartThingsPublic into MSA-635-1

egosselin69 committed:
Squash several Git commit

egosselin69 committed:
Merge branch 'master' into MSA-635-1

EricG66 commented:
@kris-schaller I hope I made it correclty. I'm really new in GitHub.

EricG66 committed:
Updated icon

EricG66 committed:
Cleanup

EricG66 committed:
cleanup

EricG66 committed:
Fix temperatureUnit issue

EricG66 committed:
Fix TemperatureUnit

EricG66 committed:
Support of batteryVoltage

EricG66 committed:
Support of batteryVoltage

EricG66 committed:
Fixed rounding temperature

EricG66 committed:
Fixes since Hub update

EricG66 committed:
Fixes since Hub update

EricG66 committed:
Removed the Farenheit functions

EricG66 committed:
Remove the farenheit code

EricG66 committed:
Remove required field

EricG66 committed:
- added swing feature
- fixes some bugs

EricG66 committed:
- added swing feature
- fixed some bugs

EricG66 committed:
Fixed icons location

EricG66 committed:
fixed icons location

EricG66 committed:
- Fixed on Farenheit support
- Fixed on swing feature
- Support some remoteCapabilities features
- Fixed some bugs

EricG66 committed:
- Fixed on Farenheit support
- Fixed on swing feature
- Support some remoteCapabilities features
- Fixed some bugs

EricG66 committed:
- Added support for remoteCapabilities
- Fixed bugs
- Added low battery notification (in beta phase)

EricG66 committed:
- Added support for remoteCapabilities
- Fixed bugs
- Added low battery notification (in beta phase)

EricG66 committed:
- Fixed bugs

EricG66 committed:
- Fixed bugs

EricG66 committed:
Fixed error on new SKY pod

EricG66 committed:
-Added new tiles for firmware and model info
-Rounding of the temperature reading

EricG66 committed:
-Added new tiles for firmware and model info
-Rounding of the temperature reading

EricG66 committed:
Fix a small bug with fan mode

EricG66 committed:
- fix one issue

EricG66 committed:
Fixed the 3rd party hosting issue with Photobucket

EricG66 committed:
Fixed the 3rd party hosting issue with Photobucket

EricG66 committed:
Activate the auto mode

EricG66 committed:
Fix some issues

EricG66 committed:
Testing a fix for Fan mode

EricG66 committed:
Testing a fix for Fan mode

EricG66 committed:
Make the app more robust when measurements cannot be returned on the Sensibo API

EricG66 committed:
Make the app more robust when measurements cannot be returned on the Sensibo API

EricG66 committed:
Add support for Google Home and Google Assistant

EricG66 committed:
Add support for Google Home and Google Assistant

EricG66 committed:
Some minor fixes

EricG66 committed:
Some minor fixes

EricG66 committed:
Rounded the humidity values to 0 decimal

EricG66 committed:
Fixed issues when some capabilities are not available.

EricG66 committed:
Add info for debugging

EricG66 committed:
fixed another issue

EricG66 committed:
Update for fanLevel when null

EricG66 committed:
Another fix

EricG66 committed:
Adding debugging infos

EricG66 committed:
Fixed fanlevel issue

EricG66 committed:
Changed the refresh of the device to 15 min.

EricG66 committed:
Some changes

EricG66 committed:
Changes for empty temperature in fan mode.

EricG66 committed:
Added support for 4 new fan level (quiet, medium_high, medium_low and strong) and 2 new swing mode (horizontal and both)

EricG66 committed:
Small fix

EricG66 committed:
Code cleanup.

EricG66 committed:
Added some custom commands to expose to webCoRE, will add more later

EricG66 committed:
Updated the log informations

EricG66 committed:
Updated the log informations
Added the refresh rate in the SmartApps

EricG66 committed:
Fixed small issue with refresh rate

EricG66 committed:
Minor fixes

EricG66 committed:
- Better usage of sensibo servers (using local storage for capabilities)
- Fixes some bugs

EricG66 committed:
- Better usage of sensibo servers (using local storage for capabilities)
- Fixes some bugs

EricG66 committed:
Small bug fix

EricG66 committed:
Cleaned up the feed notifications

EricG66 committed:
Fixed a bug in the getCapabilities function

EricG66 committed:
Added fan commands (lowfan, mediumfan, highfan, quietfan, strongfan ,autofan) for Webcore

EricG66 committed:
- Removed "mode" event and replaced it by "currentmode"

EricG66 committed:
- Removed "mode" event and replaced it by "currentmode"

EricG66 committed:
- Add support for airConditionerMode capability

EricG66 committed:
Updated version 1.6 (Beta)
This version include some fixes and add the support for Climate React

EricG66 committed:
Updated version 1.6 (Beta)
This version include some fixes and add the support for Climate React

EricG66 committed:
Updated comments

EricG66 committed:
Fixed bug in Climate React

EricG66 committed:
Fixed bug in Climate React

EricG66 committed:
Fixe a typo in a command name

EricG66 committed:
Fixed a problem with Farenheit for Climate React

EricG66 committed:
Updated the app icon to the new model

EricG66 committed:
Updated configureClimateReact comments

EricG66 committed:
Fixed issue with the Fan/Dry mode (when there is no fan levels)

EricG66 committed:
Fixed issue with the Fan/Dry mode (when there is no fan levels)

EricG66 committed:
Fixed an issue with FanLevels

EricG66 committed:
Made modifications to fix Dry and Fan mode for certains AC units

EricG66 committed:
Made modifications to fix Dry and Fan mode for certains AC units

EricG66 committed:
Fix a error in code

EricG66 committed:
Test
